pkg/lso/localvolumediscovery.go

```diff
@@ -170,64 +167,72 @@ func (builder *LocalVolumeDiscoveryBuilder) Delete() error {
 		builder.Definition.Name, builder.Definition.Namespace)
 
 	if !builder.Exists() {
-		return fmt.Errorf("localVolumeDiscovery cannot be deleted because it does not exist")
+		return nil
```

Comment:
Please add log message above:
glog.V(100).Infof("localVolumeDiscovery %s in namespace %s cannot be deleted because it does not exist",
			builder.Definition.Name, builder.Definition.Namespace)